### PR TITLE
Fix Gemfile, fix asset pipeline

### PR DESCRIPTION
**Pull request by soojison**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,10 +1,8 @@
 source 'https://rubygems.org/'
 
-gem 'rails', '4.1.0'
+gem 'rails', '4.2.5.1'
 
-# Bundle edge Rails instead:
-# gem 'rails', :git => 'git://github.com/rails/rails.git'
-# 
+gem 'activeresource' # https://github.com/rails/activeresource
```
Comment (schmitzc) [inline Gemfile]: :question: Is `ActiveResource` being used? It is used for connecting to web services, and I don't think Boardbank is doing anything like that.

File: config/application.rb [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
-# require "rails/test_unit/railtie"
+require 'rails/all'
 
 if defined?(Bundler)
   # If you precompile assets before deploying to production, use this line
-  # Bundler.require(*Rails.groups(:assets => %w(development test)))
-  #Bundler.require(:default, :assets, Rails.env)
-  Bundler.require(*Rails.groups)
+  #Bundler.require(*Rails.groups(:assets => %w(development test)))
+  Bundler.require(:default, :assets, Rails.env)
```
Comment (schmitzc) [inline config/application.rb]: :question: It looks like this is loading the `assets` group from the Gemfile, but that has been removed. In a newly created Rails 4.2.x application, this is the line included: `Bundler.require(*Rails.groups)` I think you want that instead.

File: Gemfile (hunk 1 above)
Comment (soojison) [inline Gemfile]: You're right. I'll get rid of that line and fix the other line!

**Comment by soojison** [comment]
Also, it says "This branch has conflicts that must be resolved" for my pull request. Is it because other people have pushed new edits to the repo and now my version and their version of some files doesn't match?

**Comment by schmitzc** [comment]
> Also, it says "This branch has conflicts that must be resolved" for my pull request. Is it because other people have pushed new edits to the repo and now my version and their version of some files doesn't match?

Yes, it looks like the commit to fix search updated the Gemfile.lock

**Pull request merged**